pkg/endpoint/policy.go

```diff
@@ -355,11 +295,7 @@ func (e *Endpoint) resolveL4Policy(owner Owner, repo *policy.Repository, c *poli
 // and a map matching which rules were successfully removed.
 // Must be called with Consumable mutex held.
 func (e *Endpoint) regenerateConsumable(owner Owner, labelsMap *identityPkg.IdentityCache,
-	repo *policy.Repository, c *policy.Consumable) (changed bool, rulesAdd, rulesRm policy.SecurityIDContexts, err error) {
```

Comment:
The function comment isn't consistent anymore. Could you remove `and a map matching which rules were successfully added/modified;  and a map matching which rules were successfully removed.` ?